ui/components/multichain/account-details/account-details-key.js

```diff
@@ -50,13 +56,22 @@ export const AccountDetailsKey = ({ accountName, onClose, privateKey }) => {
           variant={TextVariant.bodySm}
           style={{ wordBreak: 'break-word' }}
         >
-          {privateKey}
+          <span onClick={() => setShowDisableSelectWarn(true)}>
```

Comment:
nit: could we reduce html bloat and increase the click area of the text slightly by moving the onClick to the wrapping `Text` component?

Reply:
https://github.com/MetaMask/metamask-extension/pull/22765/commits/d531a6715697b8ef22bfbd825a22804a2c3874ea